skiko/build.gradle.kts

```diff
@@ -376,7 +383,24 @@ kotlin {
             }
             generateVersion(OS.Wasm, Arch.Wasm)
 
+            val main by compilations.getting
             val test by compilations.getting
+
+            main.compileTaskProvider.configure {
+                outputs.file(setupMjs)
+            }
+
+            test.compileTaskProvider.configure {
+                outputs.file(skikoTestMjs)
+            }
+
+            listOf(main, test).forEach {
```

Comment:
Do we need this piece at all?

Reply:
I rebased on master, therefore this comment doesn't link anymore to the code.
This part is indeed needed:
`KotlinCompilerPluginSupportPlugin` has a method `fun getPluginArtifact()` which we override.  We return `SubpluginArtifact(SkikoArtifacts.groupId, IMPORT_GENERATOR)` with null version. Therefore by default it tries to get the same version as koltin version. 

Since our compiler plugin is not published, we need to substitute this implicit dependency.

Reply:
Also I moved it to buildSrc